reana_db/models.py

```diff
@@ -286,6 +286,58 @@ class JobStatus(enum.Enum):
     queued = 5
 
 
+class UserWorkflowSession(Base, Timestamp):
+    """User Workflow Session table."""
+
+    __tablename__ = "user_workflow_session"
+    __table_args__ = {"schema": "__reana"}
+
+    user_id = Column(UUIDType, ForeignKey("__reana.user_.id_"), nullable=False)
```

Comment:
Do we need to add the user here? I rather see this table as `WorkflowSession` or `WorkflowInteractiveSession` and the relation to `User` to be simply backtracked from `InteractiveSession` through `owner_id` (I see ternary relationship was mentioned in the original issue, but I have second thoughts).

Reply:
That's a good solution IMO. @tiborsimko , what do you think?

Reply:
Yes, for this sprint and for MVP, it seems perfectly OK indeed. 

(However, we may need to refactor this later, when we shall be addressing user group and access rights. Example: currently, a `Workflow` row has an `owner_id`, but in reality there can be several people associated with a workflow: one main owner, 2-3 collaborators having some R/W rights e.g. to rerun it, a convener having R/O rights to see the results, etc. Some collaborators could run different notebooks on the same workspace, not being owners strictly speaking... These kinds of considerations can naturally wait for later; however it is good to have a future outlook with a flexible "user-workflow-session-role" structure in mind to see where we shall be heading later.)

Reply:
@tiborsimko, should we create an issue from your comment? Or this is something to bear in mind?